javascript_nodejs/2.echobot-with-counter/index.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+ 
+const path = require('path');
+const restify = require('restify');
+
+// Import reuqired bot services. See https://ama.ms/bot-services to learn more about the different part of a bot
+const { BotFrameworkAdapter, MemoryStorage, ConversationState } = require('botbuilder');
+
+// the bot's main (and only in this example) dialog
+const MainDialog = require('./dialogs/mainDialog');
+
+// Import required bot confuguration.
+const { BotConfiguration } = require('botframework-config');
+
+// Read botFilePath and botFileSecret from .env file
+// Note: Ensure you have a .env file and include botFilePath and botFileSecret.
+const ENV_FILE = path.join(__dirname, '.env');
+const env = require('dotenv').config({path: ENV_FILE});
+
+// Create HTTP server
+let server = restify.createServer();
+server.listen(process.env.port || process.env.PORT || 3978, function () {
+    console.log(`\n${server.name} listening to ${server.url}`);
+    console.log(`\nGet emulator: https://aka.ms/botframework-emulator`);
+    // TODO: Add emulator deep link
```

Comment:
Do we have a deep link for emulator?  if not, we'll need to open a ticket to track that we will need to change this.  I'm also concerned that this will get replicated in all the other nodejs samples.